### PR TITLE
created a separate landing page template, header and layout file

### PR DESCRIPTION
**Pull request by axellabs**
consolidated the two separate header partials, hid menu on landing page sub-templates

### CODE REVIEW

File: src/templates/partials/header.html [hunk 1]
```diff
@@ -7,37 +7,40 @@
...
-          It's hidden unless in a tablet breakpoint.  #}
-          {% module "site_logo" path="@hubspot/logo", label="Logo" %}
-        </div>
-        {% if content.translated_content|length %}
-        <div class="header__language-switcher header--element">
-          <div class="header__language-switcher--label">
-            {% module "language-switcher" path="@hubspot/language_switcher", display_mode='localized', label="Language Switcher" %}
-            <div class="header__language-switcher--label-current"> {{ locale_name(locale) }}</div>
+    {% if (content.subcategory != "landing_page") %}
+    {% this conditional removes the menu and other components from landing page templates %}
```
Comment (brandenrodgers) [inline src/templates/partials/header.html]: I think this should be a comment block

Comment (axellabs) [reply src/templates/partials/header.html]: ! good catch. let me fix that

**Comment by gcorne** [comment]
Instead of creating another base template could we add a header block?

**Comment by ckconant** [comment]
@gcorne can you unpack that further? Would we need a conditional for landing page & website page templates? If so, how could we build that to scale without having to manually add each template path to the conditional since we don't differentiate between landing page & website page templates currently?

**Comment by gcorne** [comment]
@ckconant I was thinking something like #70 (untested).

**Comment by axellabs** [comment]
Closed in lieu of #70

**Pull request closed without merging**